spec/models/mail_server_log_spec.rb

```diff
@@ -354,19 +389,31 @@
           allow(AlaveteliConfiguration).to receive(:mta_log_type).and_return('postfix')
         end
 
+        let(:line) do
+          "Oct 10 16:58:38 kedumba postfix/smtp[26358]: A664436F218D: " \
+          "to=<contact@openaustraliafoundation.org.au>, " \
+          "relay=aspmx.l.google.com[74.125.25.26]:25, delay=2.7, " \
+          "delays=0.16/0.02/1.8/0.67, dsn=2.0.0, " \
+          "status=sent (250 2.0.0 OK 1349848723 e6si653316paw.346)"
+        end
+
         it 'returns a delivery status for the log line' do
-          line = "Oct 10 16:58:38 kedumba postfix/smtp[26358]: A664436F218D: " \
-                 "to=<contact@openaustraliafoundation.org.au>, " \
-                 "relay=aspmx.l.google.com[74.125.25.26]:25, delay=2.7, " \
-                 "delays=0.16/0.02/1.8/0.67, dsn=2.0.0, " \
-                 "status=sent (250 2.0.0 OK 1349848723 e6si653316paw.346)"
-          log = MailServerLog.new(:line => line)
+          log = MailServerLog.new(:line => line, :order => 1)
           status = MailServerLog::PostfixDeliveryStatus.new(:sent)
           expect(log.delivery_status).to eq(status)
         end
 
+        it 'stores the delivery status value' do
+          log = MailServerLog.new(:line => line, :order => 1)
+          status = MailServerLog::PostfixDeliveryStatus.new(:sent)
+          log.delivery_status
+          expect(log.persisted?).to eq true
+        end
```

Comment:
same comments as the exim version https://github.com/mysociety/alaveteli/pull/3476/commits/c4f7c2503333adb1101ad8a58538467586d4e6a4#r78357600